tests/dm_syncer/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+TASK_NAME="test"
+
+function run() {
+    run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1
+    check_contains 'Query OK, 2 rows affected'
+    run_sql_file $cur/data/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2
+    check_contains 'Query OK, 3 rows affected'
+
+    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+    run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+    check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER2_PORT
+    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+    check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
+
+    # start a task in `full` mode
+    cat $cur/conf/dm-task.yaml > $WORK_DIR/dm-task.yaml
+    sed -i "s/task-mode-placeholder/full/g" $WORK_DIR/dm-task.yaml
+    # avoid cannot unmarshal !!str `binlog-...` into uint32 error
+    sed -i "s/binlog-pos-placeholder-1/4/g" $WORK_DIR/dm-task.yaml
+    sed -i "s/binlog-pos-placeholder-2/4/g" $WORK_DIR/dm-task.yaml
```

Comment:
How about setting these values directly in _dm-task.yaml_?

Reply:
OK